scripts/scp_api.py

```diff
@@ -99,6 +99,12 @@
 cmdline = Commandline.Commandline()
 
 
+def exists_in_bucket(bucket_file_path, mute=False):
```

Comment:
Why is this method implemented twice, but once as a `@staticmethod`?  If they have different functions (one uses `gsutil stat` and another uses `gsutil ls`), then I don't understand why they would have the same method name.

Reply:
Whoops! Eno helped me refactor so my code to make exists_in_bucket a staticmethod in the Class - didn't realize I had neglected to delete the original...